### PR TITLE
[MOSS] beta ui

### PR DESCRIPTION
**Pull request by artursmirnov**
Issue: https://github.com/travis-pro/team-teal/issues/2903

### CODE REVIEW

**Comment by artursmirnov** [comment]
@sorta thanks for the review! I have pushed a bit more changes with feedback implementation. Could you please take a look once you have some time?

File: app/models/owner.js [hunk 1]
```diff
@@ -63,25 +64,23 @@ export default Model.extend({
...
-  migrationBetaRequests: computed(function () {
-    return this.store.peekAll('beta-migration-request').filter(request =>
-      request.owner === this || request.organizations.includes(this)
-    );
-  }).volatile(),
+  fetchBetaMigrationRequestsTask: task(function* () {
+    const data = yield this.ajax.getV3(`/user/${this.accounts.user.id}/beta_migration_requests`);
+    this.store.pushPayload('beta-migration-request', data);
+    return this.store.peekAll('beta-migration-request');
+  }),
```
Comment (sorta) [inline app/models/owner.js]: Nice, I think switching this to an ember-concurrency task makes a lot of sense :+1:

Comment (artursmirnov) [reply app/models/owner.js]: Yeah, `volatile` properties don't recalculate when `store` changes, so it caused issues. It was a weird decision from the beginning 😆

**Comment by artursmirnov** [comment]
@sorta thanks for review!

File: app/models/owner.js (hunk 1 above)
Comment (sorta) [reply app/models/owner.js]: Haha yeah I've been there for sure. You write something in a way that makes sense in the moment, and then later look back and go "Why on Earth did I do it this way?" :laughing:

**Comment by artursmirnov** [comment]
Hey @sorta, we're going to release this PR soon, could you please take another look to it once you have some time?

**Comment by sorta** [comment]
Absolutely can do @artursmirnov! I'll take a look at this

File: app/services/flashes.js [hunk 2]
```diff
@@ -55,33 +55,18 @@ export default Service.extend({
...
+      const type = Object.keys(flash)[0];
+      const { message, preamble = messageTypeToPreamble[type], aboveOverlay } = flash[type];
+      const icon = messageTypeToIcon[type];
+      const closeButton = messageTypeToCloseButton[type];
+      const item = { type, message, icon, preamble, closeButton, aboveOverlay };
+
+      this.flashes.unshiftObject(item);
+
+      if (!closeButton) this.removeFlash(item);
+    });
```
Comment (sorta) [inline app/services/flashes.js]: This is much nicer, I like it! :+1:

**Pull request merged**